gui/autochop.lua

```diff
@@ -222,7 +232,7 @@ function Autochop:init()
             visible=is_not_minimal,
         },
         widgets.HotkeyLabel{
-            frame={b=8, l=0},
+            frame={b=7, l=0},
```

Comment:
to keep the current spacing, could you move these widgets back up one and change the List frame to `b=15`? You can then change the Autochop window frame to `h=28` to compensate.